index.js

```diff
@@ -107,55 +113,93 @@ exports = module.exports = function serveIndex(root, options){
     var dir = decodeURIComponent(url.pathname);
     var originalDir = decodeURIComponent(originalUrl.pathname);
 
-    // join / normalize from root dir
-    var path = normalize(join(root, dir));
+    // determine ".." display
+    var showUp = dir !== '/';
```

Comment:
This new `showUp` logic seems different. Before we only show up if actually going up will stay within `serve-index`. This will now show up regardless and let people jump out of `serve-index`.